README.md

```diff
@@ -193,6 +194,11 @@ setInterval(() => {
   trigger.trigger(10, 1); // Set trigger high for 10 microseconds
 }, 1000);
 ```
+#### Determine Current Tick
+```js
+    const Gpio = require('pigpio').Gpio;
+    var current_tick = Gpio.tick(); //this is unsigned 32bit 
```

Comment:
The code here should not be indented by four spaces. It doesn't need to be indented.

Reply:
Fixed in f716731